.github/workflows/release.yaml

```diff
@@ -112,6 +112,15 @@ jobs:
               chainloop attestation add --name ${BINARY_NAME} --value ${BINARY_PATH} || true
             done
 
+      - name: Add Attestation Artifacts (source code)
+        run: |
+          # When the trigger of the action is a release, github.ref contains refs/tags/<tag_name>
+          # Reference: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
+
+          tag=$(echo -n ${{github.ref}} | cut -d / -f3)
+          curl -sL -o /tmp/source-code.tar.gz "https://github.com/chainloop-dev/chainloop/archive/refs/tags/$tag.tar.gz"
+          chainloop attestation add --name source-code --value /tmp/source-code.tar.gz
```

Comment:
Interesting, it would be great to keep the file format too.

We could use the `gh` CLI, which I think it's installed in the runner, and as long as you have `permissions: read` in the job you should be able to use it.

For example, 

You can download the release source code by doing 
```
gh release download v0.87.1 -A tar.gz 
=> chainloop-0.87.1.tar.gz
```

In the future, we could basically download all the assets and attest them in chainloop.

```
$ gh release download v0.87.1 -D test 
$ ls test
cas.cyclonedx.json                            chainloop-cli-0.87.1-darwin-arm64.tar.gz      chainloop-cli-0.87.1-linux-amd64.tar.gz.sig  checksums.txt                cosign.pub
chainloop-cli-0.87.1-darwin-amd64.tar.gz      chainloop-cli-0.87.1-darwin-arm64.tar.gz.sig  chainloop-cli-0.87.1-linux-arm64.tar.gz      checksums.txt.sig
chainloop-cli-0.87.1-darwin-amd64.tar.gz.sig  chainloop-cli-0.87.1-linux-amd64.tar.gz       chainloop-cli-0.87.1-linux-arm64.tar.gz.sig  controlplane.cyclonedx.json
```

Reply:
That's cool, I've added the needed changes and it's using now the GitHub CLI.